backend/test_optimization.py

```
@@ -0,0 +1,119 @@
+            print(f"Compression Ratio: {(1 - compressed_size/size)*100:.1f}%")
+            print(f"Content-Encoding: {response.headers.get('Content-Encoding', 'none')}")
+            
+            # Show sample of response structure
+            if "output" in data:
+                output = data["output"]
+                if isinstance(output, list) and len(output) > 0:
+                    print("\nSample Question Fields:")
+                    print(f"  {list(output[0].keys())}")  
+            elif "output_mcq" in data:
+                # Handle combined endpoint response
+                print(f"\nCombined Response Structure:")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)